lib/commands/general.js

```diff
@@ -27,7 +27,11 @@ commands.getDeviceTime = async function () {
       stdout = stdout.trim();
       try {
         let date = new Date(stdout);
-        return date.toString();
+        let dateStr = date.toString();
+        if (dateStr === 'Invalid Date') {
+          throw new Error('Unable to parse idevicedate output');
```

Comment:
should we also print the actual output in the error message?

Reply:
It's returned (and printed) in the `catch` block.